### DISCUSSION TITLE
Support TSD (The Spaghetti Detective)

### DISCUSSION THREAD

**Discussion by NotExpectedYet** [OctoFarm/OctoFarm#206] [Feature Request]
Support for the Spaghettie detective. 

I don't currently use this so would be interested in seeing screenshots / discussions on what exactly to pull from TSD and display in OctoFarm.

**Comment by JFSolution-old** [comment]
<img width="264" alt="grafik" src="https://user-images.githubusercontent.com/26405639/104120559-0c072180-5338-11eb-8dab-9a66a166d4fa.png">
It is a stream or images with markers for anomalys and a meter to see AI result if the print failed.
So I think it would be a must have to see this meter and the images are just nice-to-have

**Reply by NotExpectedYet** [reply]
That's perfect, think I can work out something with this. Maybe do some capturing of it in history.
